### ISSUE TITLE
quickJump : Restriction to current server suggests creating new subfolder unexpectedly

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/QuickFolders#140]
I found a bug relating to issue #118 - when I activate the restriction to the current server I still see suggestions to create new subfolders on other servers if only parent (no child) matches are found. In this example I am trying to move a mail from "Rakuten TV" and create a new folder rakuten:

![image](https://user-images.githubusercontent.com/6314674/115108481-9af38d80-9f68-11eb-8e9f-233cbb65bc07.png)

If there is an existing folder match it _does not_ display the invalid option of creating a new folder across in Local Folders:

![image](https://user-images.githubusercontent.com/6314674/115108490-aa72d680-9f68-11eb-99bf-f6c0829e256d.png)

**Comment by RealRaven2000** [comment]
quick fix version:

[QuickFolders-wx-5.5pre74.zip](https://github.com/RealRaven2000/QuickFolders/files/6329020/QuickFolders-wx-5.5pre74.zip)

patch content: I had to add  a truthy second paremeter to the folder iterator of `QuickFolders.Interface.findFolderName()` :

```
for (let folder of util.allFoldersIterator(isFiling, isQuickJump)) {
  addIfMatch(folder, matches.parentString || parentString, parents);
}
```

**Comment by RealRaven2000** [comment]
Fixed in 5.5  published today
